x-pack/plugins/fleet/server/services/security/uninstall_token_service/index.ts

```diff
@@ -170,11 +175,13 @@ export class UninstallTokenService implements UninstallTokenServiceInterface {
   public async getTokenMetadata(
     policyIdFilter?: string,
     page = 1,
-    perPage = 20
+    perPage = 20,
+    excludeManagedPolicies?: boolean
```

Comment:
It would be more extensible if we make this an `excludedPolicyIds` so if we need to exclude policies in the future that aren't exclusively `is_managed = true`, we won't have to make another change here. it also separates out the logic more as the `is_managed` flag is an agent policy concern and not a uninstall token concern.

we can then get the managed policy ids in the handler and pass them in as `excludedPolicyIds`.